### ISSUE TITLE
SysUI crash after turning the phone for 1 minute (then crashloop)

### ISSUE THREAD

**Issue by cab13140** [wasdennnoch/AndroidN-ify#1147]
Caused by:  ClassCastExeption - FrameLayout$LayoutParams cannot be cast to LinearLayout$LayoutParams

Full logcat stacktrace here
```
--------- beginning of crash

11-05 15:59:03.680  4187  4187 E AndroidRuntime: FATAL EXCEPTION: main

11-05 15:59:03.680  4187  4187 E AndroidRuntime: Process: com.android.systemui, PID: 4187

11-05 15:59:03.680  4187  4187 E AndroidRuntime: java.lang.ClassCastException: android.widget.FrameLayout$LayoutParams cannot be cast to android.widget.LinearLayout$LayoutParams

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.widget.LinearLayout.measureHorizontal(LinearLayout.java:1065)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.widget.LinearLayout.onMeasure(LinearLayout.java:632)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.View.measure(View.java:18880)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at com.android.systemui.statusbar.ExpandableView.onMeasure(ExpandableView.java)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.View.measure(View.java:18880)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewGroup.measureChild(ViewGroup.java:5923)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewGroup.measureChildren(ViewGroup.java:5900)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at com.android.systemui.statusbar.stack.NotificationStackScrollLayout.onMeasure(NotificationStackScrollLayout.java)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.View.measure(View.java:18880)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5952)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.View.measure(View.java:18880)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5952)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.View.measure(View.java:18880)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5952)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.View.measure(View.java:18880)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5952)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.View.measure(View.java:18880)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2104)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1220)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1456)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1111)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6017)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:670)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:606)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:739)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:234)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:5526)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)

11-05 15:59:03.680  4187  4187 E AndroidRuntime: 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:102)

```

or [here](http://pastebin.com/Cxj95g6E) (pastebin)

# SYSTEM INFO
ROM: ROMX
Based of stock Xperia ROM
DEVICE: Xperia Z5 E6653
ANDROID VERSION: 6.0.1
Problems appears from Snashot n°726

If you need SystemUI apk, just ask ;)

**Comment by wasdennnoch** [comment]
Please post your Xposed log too.

**Comment by wasdennnoch** [comment]
Should be fixed with 0a90df8c876f6bbc3e83f0d2829f6310b7038bbd
